source/dub/packagemanager.d

```diff
@@ -271,7 +271,8 @@ class PackageManager {
 				foreach (ovr; repo.overrides)
 					if (ovr.package_ == name.toString() && ovr.source.matches(ver)) {
```

Comment:
I guess the overrides should only apply to base packages as a whole, not on a sub-package basis. I.e., `ovr.package_ == name.main.toString()`.